great_expectations/expectations/metrics/util.py

```diff
@@ -159,6 +166,20 @@ def get_dialect_regex_expression(column, regex, dialect, positive=True):
         )
         pass
 
+    try:
+        # Dremio
+        # if issubclass(dialect.dialect, sqlalchemy_dremio.pyodbc.dialect):
```

Comment:
Should this comment be here?

Reply:
@talagluck The previous Teradata connector merge caused the merge conflicts. I think there was even more broken than just the comment. The auto merge also consolidated a 'try' and 'pass'. I have fixed that one as well and the black lint.